test/openAPI/features/data_update.feature

```diff
@@ -1,17 +1,25 @@
-Feature: API endpoint that allows users to update a new record in the Digital Registries database.
-  Request endpoint: PUT /data/{registryname}/{versionnumber}/update
+@method=GET @endpoint=/data/{registryname}/{versionnumber}/update
+Feature: API endpoint that allows users to update a new record in database.
 
-  Scenario: The user successfully updates the record in the Digital Registries database
-    Given The user wants to update a record in the Digital Registries database and a record exists
-    When The user sends a valid request to update the record in the database
-    Then The operation to update a record is completed successfully
+  @smoke
+    Scenario: Successfully update user information record in database smoke type test
```

Comment:
smoke should also take all params required for "query" and "write" in the payload